airbyte-workers/src/main/java/io/airbyte/workers/temporal/scheduling/shared/ActivityConfiguration.java

```diff
@@ -26,4 +26,11 @@
       .setRetryOptions(TemporalUtils.NO_RETRY)
       .build();
 
+  public static final ActivityOptions LONG_RUN_OPTIONS = ActivityOptions.newBuilder()
+      .setScheduleToCloseTimeout(Duration.ofDays(MAX_SYNC_TIMEOUT_DAYS))
```

Comment:
I think we want this to be start to close, not schedule to close.

Reply:
It s propagated to all the other arguments. Will make it explicit.